karma.conf.js

```diff
@@ -131,10 +137,14 @@ module.exports = function(config) {
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
     browsers: [
-     'Safari',
-     'Firefox',
-     'ChromeCanary'],
-
+<<<<<<< HEAD
+=======
+     //'Safari',
+     //'Firefox',
+     //'ChromeCanary'],
+>>>>>>> 41f3d69161178ea150f292ed17384b67f2d0645a
```

Comment:
I think this is a git merge marker that got left in and overlooked.

Reply:
Removed